docs/CODEOWNERS

```diff
@@ -0,0 +1,138 @@
+# Archipelago World Code Owners / Maintainers File
+#
+# This file is used to notate who the current "owners" or "maintainers" of any currently merged world folder. For any
+# pull requests that modify contents of these worlds, a code owner must approve the PR in addition to a core maintainer.
+# This is not to be used for files/folders outside the /world folder, those will always need sign off from a core
+# maintainer. All usernames must be GitHub usernames (and are case sensitive).
+
+# Adventure
+/worlds/adventure/ @JusticePS
+
+# A Link to the Past
+/worlds/alttp/ @Berserker66
+
+# ArchipIDLE
+/worlds/archipidle/ @LegendaryLinux
+
+# Sudoku (BK Sudoku)
+/worlds/bk_sudoku/ @Jarno458
+
+# Blasphemous
+/worlds/blasphemous/ @TRPG0
```

Comment:
I know Damocles is listed as a maintainer on our list on discord but unsure what their github username would be, if they have one, as they don't show up in the commit history for the Blasphemous world folder

Reply:
Need their github username. :P

Reply:
Done.